catalog/app/utils/dragging.ts

```diff
@@ -3,17 +3,20 @@ import * as React from 'react'
 export default function useDragging() {
   const [isDragActive, setDragActive] = React.useState(false)
 
+  const setActive = React.useCallback(() => setDragActive(true), [])
+  const setInactive = React.useCallback(() => setDragActive(false), [])
```

Comment:
doesnt eslint complain about non-exhaustive dependencies? i know that setState is stable and can be omitted from dep list, but i thought eslint doesnt know this

Reply:
Eslint knows this